### PR TITLE
Updated Guam locality records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/690.

This PR updated all locality records in Guam. The following edits were made:

- All point records were given a `edtf:cessation` or `edtf:deprecated` date and `mz:is_current` property. All point records either matched an existing record with a polygon geometry or were not a valid locality (several generic "Guam Island" records).
- Superseded the point records into their appropriate polygon record
- Gave `superseded_by` property to the appropriate records
- Ran through the exportify tool

This PR should leave WOF with _only_ polygon records that match to the US Census' "places" file for Guam. This PR should not require PIPing.

### CODE REVIEW

File: data/101/734/545/101734545.geojson [hunk 1]
```diff
@@ -54,13 +54,15 @@
...
-    "wof:lastmodified":1473197727,
+    "wof:lastmodified":1490815253,
     "wof:name":"Dededo",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
     "wof:repo":"whosonfirst-data",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        890449855
```
Comment (nvkelso) [inline data/101/734/545/101734545.geojson]: I see 2 records being superseded into this one. The missing one is `890436029`.

File: data/101/734/657/101734657.geojson [hunk 2]
```diff
@@ -68,13 +68,15 @@
...
-    "wof:lastmodified":1467184489,
+    "wof:lastmodified":1490815253,
     "wof:name":"Asan",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
     "wof:repo":"whosonfirst-data",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        890436037
```
Comment (nvkelso) [inline data/101/734/657/101734657.geojson]: Why not `101734565`?

**Comment by nvkelso** [comment]
@thisisaaronland In some cases we're finding that a single WOF record technically has concordances with 2 or more GeoNames.org (and other sources) records.

For example, this primary record:

- data/101/734/549/101734549.geojson
- https://github.com/whosonfirst-data/whosonfirst-data/pull/710/files#diff-5c02f0420f894cf60d85f3f31e3ddf3a
- `"gn:id":4044012,`
- lacks a population

Supersedes: 

- data/890/414/143/890414143.geojson
- `"gn:id":"7874008",`
- has a population

We're noticing there is a "duplicate" in our data (that we're cleaning up), and we see that other sources also have a duplicate. How do we store knowledge of the duplicate in the concordances list so we don't re-introduce the external dup on later imports? 

@stepps00 let's preserve the population on the primary record in the `wof:population` property, source geonames.

**Comment by nvkelso** [comment]
@stepps00 please go thru all the superseded records to make sure the new record they point at has a population value, when it doesn't add in wof:population with the superseded value.

**Comment by stepps00** [comment]
Per https://github.com/whosonfirst-data/whosonfirst-data/pull/710#issuecomment-301621331, after reviewing the locality records we have for Guam, it makes more and more sense to come up with a solution to records that actually have more than a single concordance value per a concordance source. 

For example, Hagatna, Guam has two equally valid concordances (each below):

```
    "wof:concordances":{
        "gn:id":"7874008",
        "gp:id":"15021088",
        "qs:id":"743596",
        "wd:id":"Q30988",
        "wk:page":"Hag\u00e5t\u00f1a, Guam"
    },
```

and 

```
    "wof:concordances":{
        "fct:id":"11558486-8f76-11e1-848f-cfd5bf3ef515",
        "fips:code":"6634750",
        "gn:id":4044012,
        "gp:id":28718593,
        "qs:id":499184,
        "wd:id":"Q30988",
        "wk:page":"Hag\u00e5t\u00f1a, Guam"
    },
```

Some of the concordance values are the same in both records, but others are not. It's worth discussing because this issue has come up before - I've filed issue https://github.com/whosonfirst-data/whosonfirst-data/issues/796 to consider property updates.

**Comment by stepps00** [comment]
@nvkelso - the most recent commits added the population properties to Guam locality records. #796 was opened to track records with multiple concordance values from a single concordance source.

I tried to keep as many valid concordances for each of the current records in this PR as possible.

File: data/101/734/553/101734553.geojson [hunk 3]
```diff
@@ -61,6 +61,7 @@
     "src:geom":"quattroshapes",
     "src:geom_alt":[],
     "src:lbl:centroid":"mapshaper",
+    "src:population":0,
```
Comment (nvkelso) [inline data/101/734/553/101734553.geojson]: This seems wrong, should be set to a string value (like `gn`) not a number.

File: data/101/734/553/101734553.geojson [hunk 4]
```diff
@@ -86,13 +87,17 @@
         }
     ],
     "wof:id":101734553,
-    "wof:lastmodified":1473197727,
+    "wof:lastmodified":1495501356,
     "wof:name":"Inarajan",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:population":0,
```
Comment (nvkelso) [inline data/101/734/553/101734553.geojson]: If we don't know the population I think we'd leave the property off (but keep the `wof:population_rank`).

File: data/101/734/567/101734567.geojson [hunk 5]
```diff
@@ -35,6 +35,7 @@
     "src:geom":"quattroshapes",
     "src:geom_alt":[],
     "src:lbl:centroid":"mapshaper",
+    "src:population":"qs",
```
Comment (nvkelso) [inline data/101/734/567/101734567.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`.

File: data/101/734/567/101734567.geojson [hunk 6]
```diff
@@ -60,13 +61,17 @@
         }
     ],
     "wof:id":101734567,
-    "wof:lastmodified":1477508269,
+    "wof:lastmodified":1495501366,
     "wof:name":"Maite",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:population":0,
```
Comment (nvkelso) [inline data/101/734/567/101734567.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`. (We should still keep `wof:population_rank`, though.)

File: data/101/734/591/101734591.geojson [hunk 7]
```diff
@@ -66,13 +67,17 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1467184488,
+    "wof:lastmodified":1495498757,
     "wof:name":"Tamuning",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:population":0,
```
Comment (nvkelso) [inline data/101/734/591/101734591.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`. (We should still keep `wof:population_rank`, though.)

File: data/101/734/593/101734593.geojson [hunk 8]
```diff
@@ -123,13 +124,17 @@
         }
     ],
     "wof:id":101734593,
-    "wof:lastmodified":1477508269,
+    "wof:lastmodified":1495501377,
     "wof:name":"Toto",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:populaiton":0,
```
Comment (nvkelso) [inline data/101/734/593/101734593.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`. (We should still keep `wof:population_rank`, though.)

File: data/101/734/595/101734595.geojson [hunk 9]
```diff
@@ -87,13 +88,17 @@
         }
     ],
     "wof:id":101734595,
-    "wof:lastmodified":1477508269,
+    "wof:lastmodified":1495501382,
     "wof:name":"Umatac",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:population":0,
```
Comment (nvkelso) [inline data/101/734/595/101734595.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`. (We should still keep `wof:population_rank`, though.)

File: data/101/734/615/101734615.geojson [hunk 10]
```diff
@@ -56,13 +57,17 @@
         }
     ],
     "wof:id":101734615,
-    "wof:lastmodified":1477508270,
+    "wof:lastmodified":1495498762,
     "wof:name":"Harmon Industrial Park",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:population":0,
```
Comment (nvkelso) [inline data/101/734/615/101734615.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`. (We should still keep `wof:population_rank`, though.)

File: data/101/734/657/101734657.geojson [hunk 11]
```diff
@@ -68,13 +69,18 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1467184489,
+    "wof:lastmodified":1495501397,
     "wof:name":"Asan",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:population":0,
```
Comment (nvkelso) [inline data/101/734/657/101734657.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`. (We should still keep `wof:population_rank`, though.)

File: data/101/734/667/101734667.geojson [hunk 12]
```diff
@@ -58,13 +59,17 @@
         }
     ],
     "wof:id":101734667,
-    "wof:lastmodified":1477508270,
+    "wof:lastmodified":1495498776,
     "wof:name":"Chalan Pago",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
+    "wof:population":0,
```
Comment (nvkelso) [inline data/101/734/667/101734667.geojson]: The pop in `qs` props is `0`, so I wouldn't expect to see anything in `wof:population` or `src:population`. (We should still keep `wof:population_rank`, though.)

File: data/890/436/019/890436019.geojson [hunk 13]
```diff
@@ -46,12 +47,16 @@
...
     ],
     "wof:id":890436019,
-    "wof:lastmodified":1477509309,
+    "wof:lastmodified":1490815253,
     "wof:name":"Tamuning-Tumon-Harmon Village",
     "wof:parent_id":85681227,
     "wof:placetype":"locality",
     "wof:repo":"whosonfirst-data",
-    "wof:superseded_by":[],
+    "wof:superseded_by":[
```
Comment (nvkelso) [inline data/890/436/019/890436019.geojson]: Oh, Census.

**Comment by stepps00** [comment]
Updated records with a `"wof:population":0,` in commit  	ef9ff00.

**Comment by nvkelso** [comment]
Over to @thisisaaronland.

**Comment by nvkelso** [comment]
Per IRL discussion:

The admin-1 "regions" of Guam should be imported as `localadmin` placetype, with the individual villages within those as `locality`. This would match our practice elsewhere, including in Puerto Rico.

**Comment by stepps00** [comment]
@nvkelso - OK to merge? Last commit fixes conflicts, includes any PIP work.

**Comment by nvkelso** [comment]
Yep

**Pull request merged**